### PR TITLE
Add simple implementation for fetching Packages from the Organisation

### LINKED ISSUE #31: Get packages from organization
Make an api endpoint where you give it an Organization id and all packages of the organization are given back.

### PR DESCRIPTION
**Pull request by DamienKusters**
Because the endpoint 'CreatePackage' does not yet take into account what user or organisation the package belongs to; It's more difficult to test this feature.

### CODE REVIEW

**Comment by NealGeilen** [comment]
> Because the endpoint 'CreatePackage' does not yet take into account what user or organisation the package belongs to; It's more difficult to test this feature.

Because it cant be tested right now. This issue will be a draft until #7 is made

File: lpr.Common/Dtos/In/PackageDtoIn.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 namespace lpr.Common.Dtos.In {
   public class PackageDtoIn
   {
+    //TODO: Do we want to parse data from the ZIP/TAR package file itself?
```
Comment (NealGeilen) [inline lpr.Common/Dtos/In/PackageDtoIn.cs:6]: For a version maybe, But for a package not

File: lpr.Data/PackageData.cs [hunk 2]
```diff
@@ -24,14 +23,36 @@ public Task<List<Package>> GetTopPackagesAsync(int amount)
       throw new NotImplementedException();
     }
 
-    public async Task<Package> CreatePackageAsync(Package newPackage) {
-      // Do we also need to create the first 'fallback' version?
+    public async Task<List<Package>> GetPackagesFromOrganisationAsync(Guid organisationId) {
+      Organisation organisation;
+      try {
```
Comment (NealGeilen) [inline lpr.Data/PackageData.cs:28]: Maybe batter to not include a Try/catch here. You can validate "organization" on "!= null" and when null throw an exception

File: lpr.UnitTests/PackageTest.cs [hunk 3]
```diff
@@ -90,46 +95,74 @@ public async void ArchivePackageAsync_Throws_ApiExecption_No_Pacakage()
...
         }
 
 
         [Fact]
         public async void CreatePackageAsync_Throws_ApiException_String_TooShort()
         {
             Package testPackage = LoopFaker.Package();
+            Account testAccount = LoopFaker.Account();
+
+            testPackage.Name = "test";
```
Comment (NealGeilen) [inline lpr.UnitTests/PackageTest.cs:139]: Just a question, Why the name change?

**Review by NealGeilen — CHANGES_REQUESTED** [review]

File: lpr.Data/PackageData.cs (hunk 2 above)
Comment (DamienKusters) [reply lpr.Data/PackageData.cs]: Changed it 👍

File: lpr.UnitTests/PackageTest.cs (hunk 3 above)
Comment (DamienKusters) [reply lpr.UnitTests/PackageTest.cs:139]: For this method (CreatePackageAsync_Throws_ApiException_String_TooShort), the change is needed.
But the method (CreatePackageAsync_Throws_ApiException_Account_Doesnt_Exist) was a copy from this method and I forgot to delete it there.

**Review by NealGeilen — APPROVED** [review]

**Review by nickonos — APPROVED** [review]

**Pull request merged**